sim/common/mop/cloaks_phase_4_54.go

```diff
@@ -0,0 +1,215 @@
+package mop
+
+import (
+	"math"
+	"time"
+
+	"github.com/wowsims/mop/sim/common/shared"
+	"github.com/wowsims/mop/sim/core"
+	"github.com/wowsims/mop/sim/core/proto"
+)
+
+type cloakConfig struct {
+	label  string
+	itemID int32
+}
+
+func init() {
+	// Xing-Ho, Breath of Yu'lon
+	// Your damaging spells have a chance to grant 100% critical strike chance for 4 sec.  (Approximately 0.62 procs per minute)
+	core.NewItemEffect(102246, func(agent core.Agent, state proto.ItemLevelState) {
+		character := agent.GetCharacter()
+		label := "Xing-Ho, Breath of Yu'lon"
+
+		aura := core.MakePermanent(character.RegisterAura(core.Aura{
+			Label: label,
+		}))
+
+		shared.RegisterIgniteEffect(&character.Unit, shared.IgniteConfig{
+			ActionID:           core.ActionID{SpellID: 146198},
+			SpellSchool:        core.SpellSchoolNature,
+			DisableCastMetrics: true,
+			DotAuraLabel:       "Essence of Yu'lon",
+			TickLength:         1 * time.Second,
+			NumberOfTicks:      4,
+			NumberAOEOfTargets: 5,
+			ParentAura:         aura,
+
+			ProcTrigger: core.ProcTrigger{
+				Name:    label,
+				Harmful: true,
+				DPM: character.NewRPPMProcManager(102246, false, core.ProcMaskSpecial|core.ProcMaskProc, core.RPPMConfig{
+					PPM: 2.61100006104,
```

Comment:
Are you sure that it should be `ProcMaskSpecial` rather than `ProcMaskSpellDamage`?